lib/importer/attach_files.rb

```diff
@@ -0,0 +1,41 @@
+module Importer
+  class AttachFiles
+    def self.run(work, files_directory, file_name)
+      AttachFiles.new(work, files_directory, file_name).run
```

Comment:
Brittle to name class here, and `new(work, files_directory, file_name).run` should work just as well.